python/pyarrow/_csv.pyx

```diff
@@ -104,14 +104,6 @@ cdef class ReadOptions(_Weakrefable):
     skip_rows : int, optional (default 0)
         The number of rows to skip before the column names (if any)
         and the CSV data.
-    skip_rows_after_names : int, optional (default 0)
```

Comment:
Maybe we could rather change the order in the actual signature, since it seems useful to have skip_rows and skip_rows_after_names together?

Reply:
That makes sense. And other places in [pyarrow](https://github.com/apache/arrow/blob/7035def5ba77218c287dccbd84ae299743379fcb/docs/source/python/csv.rst) and [libarrow](https://github.com/apache/arrow/blob/7035def5ba77218c287dccbd84ae299743379fcb/cpp/src/arrow/csv/options.h#L151-L155) that reference both have them grouped together. Changed in https://github.com/apache/arrow/pull/13983/commits/44acfb34e5efad81726b5852da7cf13621065097. Thanks.